source/index.html.haml

```diff
@@ -4,7 +4,9 @@ description: Adarsh Pandit's Blog - Ruby, Rails, JavaScript, Detroit, Life
 ---
 
 %h1
+  =link_to('←', 'http://gabebw.com', title: 'Yes, this is a webring')
```

Comment:
OK this is so nitpicky:

I have a link _back_ to you, so can your _forward_ link be to me, for parity?